x/supplier/keeper/msg_server_stake_supplier_test.go

```diff
@@ -45,9 +45,12 @@ func TestMsgServer_StakeSupplier_SuccessfulCreateAndUpdate(t *testing.T) {
 	require.Equal(t, "svcId", foundSupplier.Services[0].ServiceId)
 	require.Len(t, foundSupplier.Services[0].Endpoints, 1)
 	require.Equal(t, "http://localhost:8080", foundSupplier.Services[0].Endpoints[0].Url)
+	// Assert that the supplier's account balance got deducted by the staking fee
+	balanceDecrease := keeper.SupplierStakingFee.Amount.Int64() + foundSupplier.Stake.Amount.Int64()
+	require.Equal(t, -balanceDecrease, supplierModuleKeepers.SupplierBalanceMap[ownerAddr])
```

Comment:
Am I understanding correctly that 1) the mock bank doesn't care about negative and 2) mock bank accounts are never "funded" (i.e. they start with zero balances)? :thinking: 

Perhaps #PUC.